bitnami/minio/README.md

```diff
@@ -589,4 +528,8 @@ Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
-limitations under the License.
\ No newline at end of file
+limitations under the License.
+
+## Warning
+
+All gateway features have been removed from Minio since upstream completely dropped this feature.
```

Comment:
I think we can add it in `Upgrade` section. I cannot see another `Warning` section used for this kind of notification.

Reply:
➕1️⃣